test_driver/transitions_perf_test.dart

```diff
@@ -250,8 +264,42 @@ void main([List<String> args = const <String>[]]) {
       await summary.writeSummaryToFile('transitions-crane', pretty: true);
     }, timeout: Timeout.none);
 
+    test('only Reply', () async {
+      if (!isTestingReplyOnly) return;
+
+      // Collect timeline data for just the Crane study.
+      final timeline = await driver.traceAction(
+        () async {
+          await runDemos(
+            ['reply@study'],
+            driver,
+            additionalActions: () async {
+              await driver.tap(replyFab); //tap compose
+              await driver.tap(replyExit); //exit compose page
+              await driver.tap(replySearch); //tap search icon
+              await driver.tap(replyExit); //exit search page
+              await driver.tap(replyEmail); //tap email card
+              await driver.tap(replyExit); //exit email card
+              await driver.tap(replyLogo); //tap reply logo
+              await driver.tap(replyLogo); //tap reply logo again
```

Comment:
mention that this will expand the drawer/rail?

Reply:
Thanks for reminding me! I forgot to test the rail. I will do that too.